examples/plot_9_plotly.py

```diff
@@ -0,0 +1,64 @@
+"""
+========================================
+Example with the plotly graphing library
+========================================
+
+sphinx-gallery supports examples made with the
+`plotly library <https://plotly.com/python/>`_. sphinx-gallery captures the
+``_repr_html_`` of plotly figure objects. Therefore you should return a figure
+object to display the figure.
+
+In order to use plotly, the ``conf.py`` of the project should include the
+following lines to select the appropriate plotly renderer::
+
+    import plotly.io as pio
+    pio.renderers.default = 'sphinx_gallery'
+
+**Optional**: the sphinx_gallery renderer of plotly will not generate png
+thumbnails. For png thumbnails, you can use instead the ``sphinx_gallery_png``
+renderer, and add ``plotly.io._sg_scraper.plotly_sg_scraper`` to the list of
+:ref:`image_scrapers`. The scraper requires to
```

Comment:
```suggestion
:ref:`image_scrapers`. The scraper requires you to
```

Reply:
Maybe you could mention here that a sphinx-gallery image scraper is included by the plotly package?